bitnami/influxdb/templates/_helpers.tpl

```diff
@@ -5,51 +5,29 @@ SPDX-License-Identifier: APACHE-2.0
 
 {{/* vim: set filetype=mustache: */}}
 
-
 {{/*
-Return the proper InfluxDB&trade; image name
+Return the proper InfluxDB&trade; Core image name
 */}}
 {{- define "influxdb.image" -}}
 {{ include "common.images.image" (dict "imageRoot" .Values.image "global" .Values.global) }}
 {{- end -}}
 
 {{/*
-Return the proper init container volume-permissions image name
+Return the proper image name (for the init container volume-permissions image)
 */}}
 {{- define "influxdb.volumePermissions.image" -}}
-{{ include "common.images.image" (dict "imageRoot" .Values.volumePermissions.image "global" .Values.global) }}
-{{- end -}}
-
-{{/*
-Return the proper gcloud-sdk image name
-*/}}
-{{- define "gcloudSdk.image" -}}
-{{ include "common.images.image" (dict "imageRoot" .Values.backup.uploadProviders.google.image "global" .Values.global) }}
-{{- end -}}
-
-{{/*
-Return the proper azure-cli image name
-*/}}
-{{- define "azureCli.image" -}}
-{{ include "common.images.image" (dict "imageRoot" .Values.backup.uploadProviders.azure.image "global" .Values.global) }}
-{{- end -}}
-
-{{/*
-Return the proper aws-cli image name
-*/}}
-{{- define "awsCli.image" -}}
-{{ include "common.images.image" (dict "imageRoot" .Values.backup.uploadProviders.aws.image "global" .Values.global) }}
+{{ include "common.images.image" (dict "imageRoot" .Values.defaultInitContainers.volumePermissions.image "global" .Values.global) }}
 {{- end -}}
 
 {{/*
 Return the proper Docker Image Registry Secret Names
 */}}
 {{- define "influxdb.imagePullSecrets" -}}
-{{ include "common.images.pullSecrets" (dict "images" (list .Values.image .Values.volumePermissions.image .Values.backup.uploadProviders.google.image .Values.backup.uploadProviders.azure.image) "global" .Values.global) }}
+{{ include "common.images.pullSecrets" (dict "images" (list .Values.image .Values.defaultInitContainers.volumePermissions.image) "global" .Values.global) }}
 {{- end -}}
 
 {{/*
-Create the name of the service account to use
+Create the name of the ServiceAccount to use
 */}}
 {{- define "influxdb.serviceAccountName" -}}
 {{- if or .Values.serviceAccount.enabled .Values.serviceAccount.create -}}
```

Comment:
serviceAccount.enabled seems to be a backwards compatibility value. Remove its occurences

Reply:
addressed at https://github.com/bitnami/charts/pull/33743/commits/859c509824da5be460e8f02a9701f49e440ae834